scrapy_webarchive/wacz.py

```diff
@@ -77,7 +84,35 @@ def cleanup_files(self, *files: str) -> None:
     def get_wacz_fname(self) -> str:
         """Generate WACZ filename based on the WARC filename."""
 
-        return f"{self.collection_name}-{get_current_timestamp()}.wacz"
+        return f"{self.collection_name}-{get_formatted_dt_string(format=TIMESTAMP_DT_FORMAT)}.wacz"
+    
+    def write_datapackage(self, zip_file: zipfile.ZipFile):
+        package_dict = {
+            "profile": "data-package",
+            "title": self.collection_name,
+            "description": "This is the web archive generated by a Scrapy extension for the " \
+                f"{self.collection_name} spider. It is mainly for scraping purposes as it does not contain " \
+                "any js/css data. Though it can be replayed as bare HTML if the site does not depend on JS.",
+            "created": get_formatted_dt_string(format=WARC_DT_FORMAT),
+            "software": f"scrapy-webarchive v{scrapy_webarchive_version}, using Scrapy v{scrapy_version}",
+            "resources": [{}],
```

Comment:
I'm missing `mainPageUrl` and `mainPageDate`, as well as `modified` (SHOULD include these properties).
And MUST include `wacz_version`.
https://specs.webrecorder.net/wacz/1.1.1/#datapackage-json

Reply:
I don't see any value in adding `modified` when we are only creating the WACZ file. We could potentially set this with the same value as the creation date, just to have the attribute in there. Do you think this will be useful?

Reply:
Yes. I'd need to read up on the specific meaning of MUST and SHOULD, but I'd include it, and yes, setting it to the same as the creation date makes sense here.